doc/tutorial/code/double.saw

```diff
@@ -3,7 +3,7 @@ double_imp <- llvm_extract l "double_imp";
 double_ref <- llvm_extract l "double_ref";
 let thm = {{ \x -> double_ref x == double_imp x }};
 
-r <- prove abc thm;
+r <- prove yices thm;
```

Comment:
Is anything checking these exercises/tutorials? @benjaminselfridge could we eventually get something like https://github.com/GaloisInc/cryptol/blob/master/cryptol/CheckExercises.hs implemented for this if not?

Reply:
Yep, there's an integration test that pulls them in. That's how I discovered that this would be a lot faster with Yices. :)